core/iwasm/interpreter/wasm_loader.c

```diff
@@ -1667,6 +1792,15 @@ load_import_section(const uint8 *buf, const uint8 *buf_end, WASMModule *module,
                     }
                     break;
 
+#if WASM_ENABLE_TAGS != 0
+                case IMPORT_KIND_TAG: /* import tags */
+                    /* it only counts the number of tags to import */
+                    module->import_tag_count++;
+                    u8 = read_uint8(p);
```

Comment:
Should add `CHECK_BUF(p, p_end, 1)` before this line, like L1753.

Reply:
Updated code according to recommendation